build/InstallAndroid.ps1

```diff
@@ -30,6 +31,9 @@ Copy-Item -Path microsoft-authentication-library-for-dotnet\build\AndroidSdkLice
 
 dir "$androidSdk\licenses"
 
-echo "installing android"
+echo "installing android 29"
 C:\Downloads\AndroidSdkTools\cmdline-tools\bin\.\sdkmanager --licenses --sdk_root="$androidSdk"
```

Comment:
does this take care of license acceptance both sdk installs?

Reply:
yes